packages/swirl-components/src/components/flip-card/flip-card.tsx

```diff
@@ -0,0 +1,44 @@
+import { Component, h, Host, Prop } from "@stencil/core";
+import classnames from "classnames";
+
+/**
+ * @slot slot - The card contents
+ */
+@Component({
+  shadow: true,
+  styleUrl: "flip-card.css",
+  tag: "flip-card",
+})
+export class FlipCard {
+  @Prop() as?: string = "div";
```

Comment:
*question*: this is the correct place as the comment above was in the generated files :) 
wouldn't it make more sense to set this to htmlTag or htmlElement? As the type of this prop is string i think semantically it would make more sense to give it more semantic meaning than "as". What do you think?

Reply:
I stole that naming from Polaris, and used it in different components as well. We could change it everywhere with breaking changes.

Reply:
No its ok. Was more a nice to have. Better be consistent.